src/app/RecordingList/ActiveRecordingsList.tsx

```diff
@@ -122,6 +130,47 @@ export const ActiveRecordingsList: React.FunctionComponent<ActiveRecordingsListP
     );
   }, []);
 
+  React.useEffect(() => {
+    addSubscription(context.notificationChannel.messages(NotificationCategory.RecordingCreated)
+      .subscribe(v => {
+        const event: RecordingNotificationEvent = v.message;
+        notifications.info('Recording Created', `${event.recording} created in target: ${event.target}`);
+        refreshRecordingList();
+      }));
+  }, []);
```

Comment:
I think the effect dependencies list (`[]` parameter) needs to include the various dependencies the lambda function has here - so that would be `context.notificationChannel`, `notifications`, and `refreshRecordingList`. Although in practice this is unlikely to mean much since all of these are references to things that won't change for the duration of the client lifecycle, it's still technically correct for code cleanliness.

Reply:
Ah I see, that's good to know. I've fixed this now